src/jdk.unsupported/share/classes/com/sun/nio/file/SensitivityWatchEventModifier.java

```diff
@@ -32,9 +32,17 @@
  * Defines the <em>sensitivity levels</em> when registering objects with a
  * watch service implementation that polls the file system.
  *
+ * @deprecated
+ * This {@code enum} is only used by the polling implementation of
+ * {@link java.nio.file.WatchService WatchService}.  The polling
+ * {@code WatchService} is used only on macOS and likely to be removed
+ * in a future release when a version based on the native file event
+ * notification facility becomes available.
```

Comment:
I agree it's time to deprecate this extension but I think the reasoning will need a few rounds to get right. As background, JDK-8285956 changed the default sensitivity from medium to high in JDK 19 so the need to bump the event (and thus reducing the polling interval) has mostly gone away. So maybe we should thinking about changing PollingWatchService it to ignore the these modifiers (like it is done with the native implementations). If we did that then it would be easy to word the deprecation text as it could just say that the modifier originally provided a hint to polling based WatchService implementations but is no longer used.

Reply:
Should the suggested change to PollingWatchService be addressed in the context of this PR or separately?

Reply:
JDK-8303413 was created to track this change (#12795).